### PR TITLE
Added error, warning, and info formats to chat_format.

### PR DESCRIPTION
**Pull request by tmercswims**
Used emoji to make nicer error and warning messages. Could also compound with box() or bold() or whatever else.

### CODE REVIEW

**Comment by tmercswims** [comment]
Added info as well. And I suppose I could go through the bot add use these where appropriate, if you like them.

**Comment by tekulvw** [comment]
I like this as is, should merge soon

File: cogs/utils/chat_formatting.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
+def error(text):
+    return ":no_entry_sign: {}".format(text)
```
Comment (Rapptz) [inline cogs/utils/chat_formatting.py:2]: Use `\N{NO ENTRY SIGN}` instead.

File: cogs/utils/chat_formatting.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+
+def warning(text):
+    return ":warning: {}".format(text)
```
Comment (Rapptz) [inline cogs/utils/chat_formatting.py:6]: Use `\N{WARNING SIGN}` instead.

File: cogs/utils/chat_formatting.py [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+
+def info(text):
+    return ":information_source: {}".format(text)
```
Comment (Rapptz) [inline cogs/utils/chat_formatting.py:10]: Use `\N{INFORMATION SOURCE}` instead.

**Comment by Twentysix26** [comment]
https://github.com/Twentysix26/Red-DiscordBot/commit/372a8aaae464f4e1e0c73625bdd1b0510b0477c9

Thanks Danny :+1:

**Pull request merged**